src/Agent.Plugins/GitCliManager.cs

```diff
@@ -124,6 +124,14 @@ public async Task LoadGitExecutionInfo(AgentTaskPluginExecutionContext context,
                 context.Output(StringUtil.Loc("UpgradeToLatestGit", recommendGitVersion, gitVersion));
             }
 
+            // git-lfs 2.7.1 contains a bug where it doesn't include extra header from git config
+            // See https://github.com/git-lfs/git-lfs/issues/3571
+            Version recommendedGitLfsVersion = new Version(2, 7, 2);
+            if (gitLfsVersion == new Version(2, 7, 1))
+            {
+                context.Output(StringUtil.Loc("UnsupportedGitLfsVersion", gitLfsVersion, recommendedGitLfsVersion));
```

Comment:
I think we only need to do this when the customer enabled LFS for checkout.
Do we want to include the issue in the message?

Reply:
Good ideas. I just implemented them.